### DISCUSSION TITLE
How to reject specific filetypes?

### DISCUSSION THREAD

**Discussion by bradwyatt** [react-dropzone/react-dropzone#1250] [Q&A]
React-dropzone has an `accept` prop, to accept specific filetypes but reject all others, like a whitelist. However, what if I want to reject one filetype (like blacklisting .exe files), but include all other filetypes? I didn't find a `reject` prop in the documentation.

**Comment by tchari** [comment]
This would be a great feature.

**Comment by rolandjitsu** [comment]
@bradwyatt I think you can use a custom validator for that (see https://react-dropzone.js.org/#section-custom-validation) in combination with accept anything (`*`).
